dags/hindawi/hindawi_params.py

```diff
@@ -7,7 +7,7 @@ def __init__(
         from_date: str = (date.today() - timedelta(days=1)).strftime("%Y-%m-%d"),
         until_date: str = date.today().strftime("%Y-%m-%d"),
         verb: str = "listrecords",
-        set: str = "HINDAWI.AA",
+        set: str = "HINDAWI.AHEP",
```

Comment:
are we getting only one set?

Reply:
I will ask Anne, I'm not sure, all possible sets:
    HINDAWI.AA (Advances in Astronomy)
    HINDAWI.AHEP (Advances in High Energy Physics)
    HINDAWI.AMP (Advances in Mathematical Physics)
    HINDAWI.JAS (Journal of Astrophysics)
    HINDAWI.JCMP (Journal of Computational Methods in Physics)
    HINDAWI.JGRAV (Journal of Gravity)

But from Anne comment on Zulip, I believe is just one:
`For Hindawi, we have only one journal, so the value for Journal name is anyway always the same.`

Reply:
We should double check that, it shouldn't be only one set here imo

Reply:
yes, it's just one. I already talked with Anne about it